src/components/Lesson/ImprovePage.scss

```diff
@@ -0,0 +1,76 @@
+@import '../../styles/variables';
+
+.link {
+  text-decoration: none !important;
+  text-align: center;
+  font-size: 16px;
+  font-weight: 350;
+
+  border-radius: 5px;
+  padding: 7px;
+
+  opacity: 0.85;
+  transition: 0.15s;
+}
+/* Css for "Rapporter et problem" */
+.newIssue{
+  composes: link;
+  background-color: #e6e6ff;
```

Comment:
Er dette en farge vi potensielt skal bruke flere steder? Hvis "ja", vurder å legge denne inn i src/styles/variables.scss. Gjelder også for de andre fargene du har brukt på denne siden. (Hvis ikke lar du det bare være).